demos/whisper-base/main.js

```diff
@@ -672,7 +672,7 @@ const main = async () => {
   try {
     context = new AudioContext({ sampleRate: kSampleRate });
     const whisper_url = location.href.includes("github.io")
-      ? "https://huggingface.co/onnxruntime-web-temp/demo/resolve/main/whisper-base/"
+      ? "https://huggingface.co/microsoft/whisper-base-webnn/resolve/main/"
```

Comment:
Curiously when I paste "https://huggingface.co/microsoft/whisper-base-webnn/resolve/main/" into the browser, I get 404? (though I get "entry not found" with the old URL too, and so something more is happening here)

Reply:
@fdwr 

We need to add detailed file names after the whisper_url, e.g.

- https://huggingface.co/microsoft/whisper-base-webnn/resolve/main/whisper_base_encoder_lm_fp16_layernorm.onnx
- https://huggingface.co/onnxruntime-web-temp/demo/resolve/main/whisper-base/whisper_base_encoder_lm_fp16_layernorm.onnx

Reply:
Yeah, we definitely need to concatenate the final filenames afterward for the full URL. I just figured there would be a directory listing or something. (resolving this lingering open tab)

Reply:
@fdwr If you want the directory list, https://huggingface.co/microsoft/whisper-base-webnn/tree/main will work,  the https://huggingface.co/microsoft/whisper-base-webnn/resolve/main/ +filenames for download or via fetch() API in JS only.

`tree` vs `resolve`

Reply:
🫡